homeassistant/components/gstreamer/media_player.py

```diff
@@ -33,6 +33,12 @@
     {vol.Optional(CONF_NAME): cv.string, vol.Optional(CONF_PIPELINE): cv.string}
 )
 
+GSP_STATE_MAPPING = {
+    STATE_IDLE: MediaPlayerState.IDLE,
+    STATE_PAUSED: MediaPlayerState.PAUSED,
+    STATE_PLAYING: MediaPlayerState.PLAYING,
+}
+
```

Comment:
This new mapping dict is based on https://github.com/happyleavesaoc/gstreamer-player/blob/61bea2c807fde0fb1c9eed6172c23e26c10573c7/gsp/__init__.py#L22-L24

Reply:
👍